src/pages/index.vue

```diff
@@ -1126,11 +1126,27 @@
     <section id="all-story-matters">
       <IndexPageHeading :text="$t('index_page_all_stories_matter_heading')" />
 
+      <div class="relative pt-[5rem] pb-[4rem]">
+        <client-only>
+          <lazy-component
+            class="absolute inset-0 -top-full pointer-events-none"
+            @show.once="fetchTrendingNFTList"
+          />
+        </client-only>
+        <IndexPageTrendingNFTList
```

Comment:
Can we lazy load this component all together? swiper is resource heavy and below the fold

Reply:
Updated